internal/manifest/crypto_test.go

```diff
@@ -96,3 +98,43 @@ func newKey(t *testing.T, bits int) *rsa.PrivateKey {
 	t.Logf("generated key: %q", HexString(x509.MarshalPKCS1PrivateKey(key)))
 	return key
 }
+
+func TestSeedShareKeyParseMarshal(t *testing.T) {
+	keyData, err := NewSeedShareOwnerPrivateKey()
+	require.NoError(t, err)
+
+	key, err := ParseSeedshareOwnerPrivateKey(keyData)
+	require.NoError(t, err)
+
+	pubHexStr := MarshalSeedShareOwnerKey(&key.PublicKey)
+
+	_, err = ParseSeedShareOwnerKey(pubHexStr)
+	require.NoError(t, err)
```

Comment:
This looks like a leftover (the function is not defined?).

Reply:
Nevermind, missed that it already existed.

Reply:
The function is defined [here](https://github.com/edgelesssys/contrast/blob/81bbc6bf5acd92a1230abfffd55e25a617ae08bf/internal/manifest/crypto.go#L75).